packages/state-transition/src/util/epoch.ts

```diff
@@ -52,7 +52,7 @@ export function computeExitEpochAndUpdateChurn(state: CachedBeaconStateElectra,
   // Exit doesn't fit in the current earliest epoch.
   if (exitBalance > exitBalanceToConsume) {
     const balanceToProcess = Number(exitBalance) - exitBalanceToConsume;
-    const additionalEpochs = Math.floor((balanceToProcess - 1) / (perEpochChurn + 1));
+    const additionalEpochs = Math.ceil((balanceToProcess - 1) / (perEpochChurn + 1));
```

Comment:
Could you detail this? Also, is it worth having some test for this, as apparently this change didn't require any associated test change.

Reply:
Ideally my preferences is to have unit tests for every function, but unfortunatley we don't have it yet. 

I updated with a code comment.

Reply:
Oh! I misread the spec [here](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#new-compute_exit_epoch_and_update_churn). 
We should do
```
const additionalEpochs = Math.floor((balanceToProcess - 1) / perEpochChurn) + 1
```

Let me know if the spec test's still passing after making this change